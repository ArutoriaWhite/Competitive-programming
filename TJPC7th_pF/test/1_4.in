42084794 2
6807800 0
20894042 1
