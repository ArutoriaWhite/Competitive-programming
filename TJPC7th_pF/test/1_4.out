6807800 14086242
